### PR TITLE
improve invite error handling

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: react/features/invite/functions.js [hunk 1]
```diff
@@ -5,6 +5,71 @@ declare var interfaceConfig: Object;
...
+ * @param {string} mucURL - In which MUC the conference exists.
+ * @returns {Promise} - The promise created by the request.
+ */
+export function getDialInConferenceID(
+        baseUrl: string,
+        roomName: string,
+        mucURL: string): Promise<Object> {
+    const conferenceIDURL = `${baseUrl}?conference=${roomName}@${mucURL}`;
+
+    return fetch(conferenceIDURL)
```
Comment (bbaldino) [inline react/features/invite/functions.js]: i think we should put the common json fetch logic in a util somewhere

File: react/features/base/util/httpUtils.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Wrapper around fetch GET requests to handle json-ifying the response
+ * and logging errors.
+ *
+ * @param {string} url - The URL to perform a GET against.
+ * @returns {Promise}
+ */
+export function doGet(url) {
```
Comment (bbaldino) [inline react/features/base/util/httpUtils.js]: should it be `doGetJson` ?

Comment (virtuacoplenny) [reply react/features/base/util/httpUtils.js]: This seems like a fair request. The response streams can be converted to a few formats, although we only really use json, so specifying json seems explicit.

Comment (virtuacoplenny) [reply react/features/base/util/httpUtils.js]: I've named it doGetJSON to be consistent with how prefixURL and prefixID and prefixURI get capitalized. I suppose I could also capitalize it GET...

**Pull request merged**